cmd/lightchain/run.go

```diff
@@ -102,7 +102,14 @@ func runCmd() *cobra.Command {
 				dbCfg.GethIpcPath(),
 			)
 
-			tracerCfg := tracer.NewConfig(shouldTrace, traceLogFilePath)
+			tracerCfg := tracer.NewConfig(shouldTrace, path.Join(dataDir, "tracer.log"))
+			if shouldTrace {
+				logger.Info("|--------")
+				logger.Info("| Danger: Tracing enabled is not recommended in production!")
+				logger.Info(fmt.Sprintf("| Tracing output is configured to be persisted at %v", tracerCfg.LogFilePath))
+				logger.Info("|--------")
+			}
+
```

Comment:
can u move this duplicated logging to a simple function in the `cmd` package?

Reply:
Done